cmd/dbfactory/cluster.go

```diff
@@ -133,6 +134,15 @@ func newSearchCommand(svc rdsiface.RDSAPI) *cobra.Command {
 				if err != nil {
 					return errors.Wrapf(err, "failed to describe DBSubnetGroup: %s", *r.DBSubnetGroup)
 				}
+				if r.DBClusterMembers != nil {
```

Comment:
Not sure if it is possible that `DBClusterMembers` will be not nil but empty, but using `if len(r.DBClusterMembers) > 0` would handle both cases.